src/init.sh

```diff
@@ -18,25 +24,136 @@ function init_kw()
     exit 0
   fi
 
-  if [[ -f "$PWD/$name" ]]; then
-    if [[ "$*" =~ --?f(orce)? || $(ask_yN "$name already exists, do you wish to overwrite it?") =~ '1' ]]; then
-      mv "$PWD/$name" "$PWD/$name.old"
+  if ! is_kernel_root "$PWD"; then
+    complain 'This command should be run in a kernel tree.'
+    exit 125 # ECANCELED
+  fi
+
+  parse_init_options "$@"
+  if [[ "$?" -gt 0 ]]; then
+    complain "${options_values['ERROR']}"
+    return 22 # EINVAL
+  fi
+
+  if [[ -f "$PWD/$KW_DIR/$name" ]]; then
+    if [[ -n "${options_values['FORCE']}" || $(ask_yN "$KW_DIR/$name already exists, do you wish to overwrite it?") =~ '1' ]]; then
```

Comment:
Maybe something like, "It looks like that you already have a kw config file. Do you want to overwrite it?"